app/Http/Controllers/ProductController.php

```diff
@@ -11,6 +12,11 @@ public function list()
         return view('product_list');
     }
 
+    public function products()
```

Comment:
```suggestion
    public function getAllProducts()
```
đặt tên function có ý nghĩa nhé

Reply:
Đã sửa ạ.